doc/decisions/backend_orm_crate.md

```diff
@@ -0,0 +1,70 @@
+# Backend ORM/SQL crate
+
+## Problem
+
+We are currently using [diesel](https://github.com/diesel-rs/diesel) as our ORM in Rust.  
+Since diesel doesn't support async out of the box we will look at other alteratives and see if a switch is reasonable.  
+There might also be advantages of other ORM crates that as of writing this have not been considered.
```

Comment:
This is not a problem (unless it is clear why we would need async). The problem below could be used: that users might use async wrongly.